app/routers/api_keys.py

```diff
@@ -74,7 +76,7 @@ async def get_api_keys(current_user: Annotated[dict, Depends(get_user_from_jwt_t
 )
 async def delete_api_key(
     api_key_id: uuid.UUID,
-    current_user: Annotated[dict, Depends(get_user_from_jwt_token)],
+    current_user: Annotated[dict, Depends(get_jwt_info_from_user_token)],
```

Comment:
For me this is a bit confusing, you expect a current_user but the name of the function is jwt_info.

Reply:
yep, previously was `get_user_from_jwt_token` but it is nopt returning an user is returning the jwt dict. 
I think that `get_jwt_info_from_user_token` would be better or maybe just `get_jwt_from_user_token` or  `get_jwt_from_auth_token`. What do you think that would be the best option?
```suggestion
    jwt_info: Annotated[dict, Depends(get_jwt_info_from_user_token)],
```